Cargo.toml

```diff
@@ -14,12 +14,12 @@ keywords = ["phenopackets", "phenotype", "ontology", "bioinformatics"]
 maintenance = { status = "experimental" }
 
 [features]
-serde = ["dep:serde"]
+serde = ["serde/derive"]
 
 [dependencies]
 prost = "0.13"
 prost-types = "0.13"
-serde = { version = "1.0.219", optional = true }
+serde = { version = "1.0.219", features = ["derive"], optional = true }
```

Comment:
I am not sure we need to add `features = ["derive"]` here, since we ask for `derive` on line 17 in `serde/derive`.

The tests run OK even if the line 22 omits the `features` tag: `serde = { version = "1.0.219", optional = true }`.